### ISSUE TITLE
HAVING statement with Upper AND Lower bounds Is treated as an OR

### ISSUE THREAD

**Issue by Jhors2** [implydata/plyql#22]
In the following query:
```
SELECT max(value) AS MAX, 
(SELECT TIME_BUCKET(__time, $interval, 'Etc/UTC') as 'Time', sum(value) AS 'Value' GROUP BY 1)
 AS TIME_RANGE FROM data WHERE '2016-04-25T15:21:00.000Z' 
<= __time AND __time < '2016-04-25T21:20:49.031Z' 
having MAX >= $LOW AND MAX <= $HIGH ORDER BY MAX DESC LIMIT 25
```
The AND syntax is treated as an or rather than an AND.  If I take off either side of the bound I get the expected result, but with both of the instances I get all results.

**Comment by vogievetsky** [comment]
Great spot. Will cut a new release when this PR goes in: https://github.com/implydata/plywood/pull/82

**Comment by vogievetsky** [comment]
Should be fixed in plyql@0.6.27
